README.md

```diff
@@ -120,3 +120,152 @@ the components involved in the test:
 ``
 cd test/integration && dlv test -- -test.run ^TestCrud$
 ``
+
+#### E2E
+
+The fnord E2E tests can run in an unmanaged, managed, or hybrid modes. For both
+unmanaged and hybrid modes, you will need to bring your own clusters (BYOC).
+The managed mode runs similarly to integration as it uses the same test fixture
+setup.
+
+##### Managed
+
+The E2E managed tests will spin up the same environment as the
+[Integration](README.md#integration) tests described above and run [CRUD
```

Comment:
It would be helpful to expand a tiny bit on what CRUD means here - do the tests ensure that resources are put into a target cluster?

Reply:
Done.